Makefile

```diff
@@ -226,12 +233,12 @@ itest: check_go_version ## Run tests iteratively (see usage for more)
 .PHONY: go_mockgen
 go_mockgen: ## Use `mockgen` to generate mocks used for testing purposes of all the modules.
 	find . -name "*_mock.go" | xargs --no-run-if-empty rm
-	go generate ./x/application/types/
-	go generate ./x/gateway/types/
-	go generate ./x/supplier/types/
-	go generate ./x/session/types/
+	# go generate ./x/application/types/
```

Comment:
This will get reverted as module migration branches get merged in to the migration base branch.

(see: https://github.com/pokt-network/poktroll/pull/372#discussion_r1488214557)